src/pages/staking/staking.tsx

```diff
@@ -165,39 +166,48 @@ const FormDelegationRequest = () => {
 
   const [t] = useTranslation();
 
-  const processValidatorList = (validatorList: ValidatorModel[] | null) => {
+  const processValidatorList = async (validatorList: ValidatorModel[] | null) => {
     if (validatorList) {
       let willDisplayWarningColumn = false;
       let displayedWarningColumn = false;
+      const apiClient = ChainIndexingAPI.init(currentSession.wallet.config.indexingUrl);
+
+      return await Promise.all(
+        validatorList.map(async (validator, idx) => {
+          if (
+            new Big(validator.cumulativeSharesIncludePercentage!).gte(
+              CUMULATIVE_SHARE_PERCENTAGE_THRESHOLD,
+            ) &&
+            !displayedWarningColumn
+          ) {
+            displayedWarningColumn = true;
+            willDisplayWarningColumn = true;
+          }
+          const [validatorApyRaw, validatorUptime] = await Promise.all([
+            apiClient.getValidatorsAverageApy([validator.validatorAddress]),
+            apiClient.getValidatorUptimeByAddress(validator.validatorAddress),
```

Comment:
```suggestion
            apiClient.getValidatorUptimeByAddress(validator.validatorAddress),
```

Definitely not a good idea to call API client hundred times here. 
Can we use a `getValidatorsUptime()` instead, which could return uptimes of an input array of validatorsAddressList?


<img width="287" alt="Screenshot 2021-11-02 at 5 12 27 PM" src="https://user-images.githubusercontent.com/74586409/139818796-bddcfb32-3ca9-475b-9364-cb4158da3f2e.png">